common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import PureComponent from 'react-pure-render/component';
+import FontAwesome from 'react-fontawesome';
+
+const propTypes = {
+  close: PropTypes.func,
+  keyMap: PropTypes.object,
+  mapKeys: PropTypes.func,
+  open: PropTypes.bool.isRequired,
+  submitChallenge: PropTypes.func.isRequired,
+  successMessage: PropTypes.string.isRequired
+};
+
+export default class ClassicModal extends PureComponent {
+
+  keyBindings(event) {
+    const map = Object.assign({}, this.props.keyMap);
+    const { open, submitChallenge, mapKeys } = this.props;
+    map[event.keyCode] = event.type === 'keydown';
+    // Ctrl + Enter
+    if (map[17] && map[13] && open) {
+      submitChallenge();
+    }
+    mapKeys(map);
+  }
+
+
+  render() {
+    let {
+      close,
+      open,
+      submitChallenge,
+      successMessage
+    } = this.props;
+    return (
+      <Modal
+        animation={ false }
+        dialogClassName='success'
+        id='complete-courseware-dialog'
```

Comment:
css should target classes not id's please fix them if they do and remove the id's